library/src/meta/heading.rs

```diff
@@ -93,6 +93,35 @@ pub struct HeadingElem {
     #[default(true)]
     pub outlined: bool,
 
+    /// Whether the heading should appear as a bookmark in the exported PDF's
+    /// outline. Doesn't affect other export formats, such as PNG.
+    ///
+    /// ```example
+    /// #outline()
+    ///
+    /// #heading[Normal heading]
+    /// Nothing changes here.
+    ///
+    /// #heading(outlined: false)[Still bookmarked]
+    /// This heading does not appear
+    /// in the Typst outline, but
+    /// will still appear in the
+    /// PDF's bookmark outline.
+    ///
+    /// #heading(bookmark: false)[Not bookmarked]
+    /// This heading still appears
+    /// in the Typst outline, but
+    /// won't be bookmarked in the
+    /// resulting PDF.
+    ///
+    /// #heading(outlined: false, bookmark: false)[Fully hidden]
+    /// This heading is hidden both
+    /// from the Typst outline and
+    /// from the resulting PDF's
+    /// bookmark outline.
```

Comment:
the closing backticks are missing. I also think that not all four combination need to be shown in the example (you can't see anything after all). If at all, the four combination would perhaps better be explained outside of the example in a bullet list. We have noticed before that people tend not to read the prose in the example itself.

Reply:
Alright, I've changed that now. :+1: